### PR TITLE
gcp: add gce deployment

### PR DESCRIPTION
**Pull request by Altonhe**
Using Opentofu to deploy the OBA server on GCP GCE

### CODE REVIEW

**Comment by aaronbrethorst** [comment]
will review soon!

File: modules/aws-ec2/main.tf [hunk 1]
```diff
@@ -143,9 +143,9 @@ resource "null_resource" "remove_ssh_keys" {
     always_run = timestamp()
   }
 
-  # replace `rm` with `del` if you are using Windows
+  # replace with `del ${path.module}\\ssh\\id_rsa ${path.module}\\ssh\\id_rsa.pub` if you are using Windows
```
Comment (aaronbrethorst) [inline modules/aws-ec2/main.tf:146]: 👍

File: modules/gcp-gce/init.tpl [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+git clone https://github.com/onebusaway/onebusaway-docker.git
+
+cd /home/${user}/onebusaway-docker
+
+# create .env file
+echo "${docker_env}" > .env
+
+
+# onebusaway-api-webapp depends on mysql, normally this will handle by docker-compose
+# but in Azure, the mysql container will not be ready when onebusaway-api-webapp starts
```
Comment (aaronbrethorst) [inline modules/gcp-gce/init.tpl]: Your comment refers to Azure, but this PR is about GCP. Should the comment be updated, or should this DB initialization stanza be removed?

**Review by aaronbrethorst — CHANGES_REQUESTED** [review]
this looks great! I have one question that I'd like to clear up before this gets merged.

File: modules/gcp-gce/init.tpl (hunk 2 above)
Comment (Altonhe) [reply modules/gcp-gce/init.tpl]: I will update the comment, MySql container sometimes acts quirky, therefore starting it first will prevent these problems.

**Review by aaronbrethorst — APPROVED** [review]
great work! let's goooo!

**Pull request merged**